src/ray/protobuf/gcs.proto

```diff
@@ -311,6 +311,11 @@ message ObjectTableDataList {
   repeated ObjectTableData items = 1;
 }
 
+message ObjectChange {
+  bool is_add = 1;
+  ObjectTableData data = 2;
+}
```

Comment:
```suggestion
// A notification message about one object's locations being changed.
message ObjectLocationChange{
  // List of node ids on which the object has appeared.
  repeated bytes added_nodes;
  // List of node ids on which the object was removed. 
  repeated bytes removed_nodes;
}
```
Maybe this will be clearer.

Reply:
fixed